gflownet/proxy/tree.py

```diff
@@ -11,21 +11,20 @@
 class TreeProxy(Proxy):
     """
     Simple decision tree proxy that uses empirical frequency of correct predictions for
-    computing likelihood, and the number of leafs in the tree for computing the prior.
+    computing likelihood, and the number of nodes in the tree for computing the prior.
     """
 
-    def __init__(self, use_prior: bool = True, **kwargs):
+    def __init__(self, use_prior: bool = True, beta: float = 1.0, **kwargs):
```

Comment:
Little docstring for `use_prior` and `beta`? :sweat_smile:

Reply:
I added something, though all of this should be re-implemented anyway.